base/float.jl

```diff
@@ -858,6 +859,21 @@ julia> floatmin(Float32)
 julia> floatmin()
 2.2250738585072014e-308
 ```
+
+# Extended help
+
+The smallest positive number without this restriction is
+[`nextfloat`](@ref)`(zero(T))`; see [`issubnormal`](@ref) for an explanation
+of sub-normal numbers.
+
+Floating point numbers always include infinity, [`Inf`](@ref), and
+negative real infinity, `-Inf`, which is what [`typemax`](@ref)`(T)`
+and [`typemin`](@ref)`(T)`, for a floating-point type `T`, would return.
+
+The smallest finite number representable by a floating-point type `T`
+is gotten by negating `floatmax` i.e. `-`[`floatmax`](@ref)`(T)`.
+
+See also: [`eps`](@ref), [`prevfloat`](@ref), [`nextfloat`](@ref).
```

Comment:
imo `typemin`, and `floatmax` are even more closely related functions. (`?floatmax` links to `floatmin` and `typemax`)

Reply:
So the `prevfloat` and `nextfloat` should be changed to `typemax` and `floatmin`?

Reply:
I like the see also section of `?floatmax` and think we should have a similar section in standard help (before extended help)

Reply:
DONE!!!